README.md

```diff
@@ -1 +1,57 @@
-# dont-panic-api
\ No newline at end of file
+# dont-panic-api [WIP]
+Work in progress - the current version doesn't have the basic features yet and shouldn't be used
+
+This is an API written to support your studies in Competitive Programming.
+Its goal is to be tool to:
+- List subjects with difficulty and dependencies
+- List problems for each subject so you can practice
+- Suggest new subjects to learn
+- Suggest new problems to solve
+- Track subjects you've learned
+- Track problems you've solved
+- Suggest when its time to review some subject by recommending problems of that subject (based on spaced repetition method)
+- Give permissions for coaches and teammates to your progress
+
+## Pre-requisites
+This API uses Azure CosmosDB graph database. To use it you will need to configure these environment variables for accessing your db:
+
+```
+$env:CDB_USERNAME="/dbs/<collection name>/<graph name>"
+$env:CDB_HOST="wss://<resource name>.gremlin.cosmos.azure.com"
+$env:CDB_KEY="<resource access key>"
+```
+
+## Install and run
+```
+go build .
+.\dont-panic-api
+```
+
+## Examples to test the API
```

Comment:
Suggestion: add example request and example response

Reply:
Vou deixar esse pra depois